stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1180,6 +1187,10 @@ public enum TestRunPredicate : CustomStringConvertible {
     case LinuxAny(reason: let reason):
       return "LinuxAny(*, reason: \(reason))"
     }
```

Comment:
hmm, it does for me, weirdly. Apparently something went wrong when I rebased and updated the pull request. Let me try again.